backend/apps/ifc_validation/checks/check_bsdd.py

```diff
@@ -1,396 +1,622 @@
 import ifcopenshell
 import logging
-import sys 
+import sys
 import requests
 import json
 import argparse
 import functools
-#from helper import database
+
 
 logger = logging.getLogger()
 
-# WORK IN PROGRESS
 
 @functools.lru_cache(maxsize=128)
-def get_classification_object(uri):
-    url = "https://bs-dd-api-prototype.azurewebsites.net/api/Classification/v3"
-    return requests.get(url, {'namespaceUri':uri})
+def find_dictionary_by_uri(uri):
+
+    url = "https://api.bsdd.buildingsmart.org/api/Dictionary/v1"
+    response = requests.get(url, {'uri': uri })
+    if response.status_code == 200:
+        json = response.json()
+        return json['dictionaries'][0] if json['count'] == 1 else None
```

Comment:
Please log or print a basic message for http error responses 4xx/5xx etc. to aid in future debugging.